scripts/index.js

```diff
@@ -1,45 +1,54 @@
+/* eslint-disable comma-dangle */
```

Comment:
- Improve your code and don't use lot of rules in your ESLint. Try to fix the issues.